scripts/deploymentUtils.ts

```diff
@@ -207,6 +207,7 @@ export async function deployAllContracts(
   )
   const rollupUser = await deployContract('RollupUserLogic', signer, [], verify)
   const upgradeExecutor = await deployUpgradeExecutor(signer)
+  await upgradeExecutor.deployTransaction.wait()
```

Comment:
@**Jeremy He**  why are we waiting for only the `UpgradeExecutor` contract to deploy and not for any other contracts

Reply:
nvm It makes sense to me now - deployContract is calling `wait` for other contracts